Packs/qualys/Integrations/Qualysv2/Qualysv2.yml

```diff
@@ -75,7 +75,7 @@ configuration:
   hidden:
   - xsoar
   - xpanse
-- additionalinfo: The fetch interval. It is recommended to set it to 24 hours, and the minimum interval is 1 hour.
+- additionalinfo: The fetch interval for assets and vulnerabilities. It is recommended to set it to 24 hours, and the minimum interval is 1 hour.
```

Comment:
```suggestion
- additionalinfo: The fetch interval for assets and vulnerabilities. It is recommended to set it to 24 hours. Interval lower then 1 hour is not supported.
```

Reply:
Updated here and synced changes to the integration `README.md`